VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
+
+Dataset
+
+Questions
+
+Theory
+
+Explanation
+
+Thesis
+
+Data
+
+Conclusions of Data
```

Comment:
Descriptive Statistics
Regression
Specific exploration ways to solve questions

Reply:
Coefficients:
               Estimate Std. Error t value Pr(>|t|)    
(Intercept)  -0.7149775  0.0486996  -14.68   <2e-16 ***
Critic_Score  0.0154849  0.0006921   22.37   <2e-16


Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept) 3.099e-01  1.072e-02   28.90   <2e-16 ***
User_Count  4.065e-04  1.835e-05   22.15   <2e-16 ***

     ...1           Name             Platform         Year_of_Release   
 Min.   :    1   Length:16719       Length:16719       Length:16719      
 1st Qu.: 4180   Class :character   Class :character   Class :character  
 Median : 8360   Mode  :character   Mode  :character   Mode  :character  
 Mean   : 8360                                                           
 3rd Qu.:12540                                                           
 Max.   :16719                                                           
                                                                         
    Genre            Publisher           NA_players        EU_players    
 Length:16719       Length:16719       Min.   : 0.0000   Min.   : 0.000  
 Class :character   Class :character   1st Qu.: 0.0000   1st Qu.: 0.000  
 Mode  :character   Mode  :character   Median : 0.0800   Median : 0.020  
                                       Mean   : 0.2633   Mean   : 0.145  
                                       3rd Qu.: 0.2400   3rd Qu.: 0.110  
                                       Max.   :41.3600   Max.   :28.960  
                                                                         
   JP_players      Other_players      Global_players     Critic_Score  
 Min.   : 0.0000   Min.   : 0.00000   Min.   : 0.0100   Min.   :13.00  
 1st Qu.: 0.0000   1st Qu.: 0.00000   1st Qu.: 0.0600   1st Qu.:60.00  
 Median : 0.0000   Median : 0.01000   Median : 0.1700   Median :71.00  
 Mean   : 0.0776   Mean   : 0.04733   Mean   : 0.5335   Mean   :68.97  
 3rd Qu.: 0.0400   3rd Qu.: 0.03000   3rd Qu.: 0.4700   3rd Qu.:79.00  
 Max.   :10.2200   Max.   :10.57000   Max.   :82.5300   Max.   :98.00  
                                                        NA's   :8582   
  Critic_Count     User_Score          User_Count       Developer        
 Min.   :  3.00   Length:16719       Min.   :    4.0   Length:16719      
 1st Qu.: 12.00   Class :character   1st Qu.:   10.0   Class :character  
 Median : 21.00   Mode  :character   Median :   24.0   Mode  :character  
 Mean   : 26.36                      Mean   :  162.2                     
 3rd Qu.: 36.00                      3rd Qu.:   81.0                     
 Max.   :113.00                      Max.   :10665.0                     
 NA's   :8582                        NA's   :9129                        
    Rating         
 Length:16719      
 Class :character

Reply:
(SQL to Video Game Manufacturer winner or most influential)